lighthouse/datadog_checks/lighthouse/lighthouse.py

```diff
@@ -0,0 +1,82 @@
+# the following try/except block will make the custom check compatible with any Agent version
+try:
+    # first, try to import the base class from old versions of the Agent...
+    from checks import AgentCheck
+except ImportError:
+    # ...if the above failed, the check is running in Agent version 6 or later
+    from datadog_checks.checks import AgentCheck
+    from datadog_checks.errors import CheckException
+
+from datadog_checks.utils.subprocess_output import get_subprocess_output
+import json
+
+# content of the special variable __version__ will be shown in the Agent status page
+__version__ = "1.0.0"
+
+
+class LighthouseCheck(AgentCheck):
+    def check(self, instance):
+        EXPECTED_RESPONSE_CODE = "NO_ERROR"
```

Comment:
Global variable should be defined outside of the class, you can put it above the class declaration.
Also, is that what Lightouse returns as response code? No HTTP code?

Reply:
Moved global variable outside the class declaration, ty.  oddly in regards to your other question, yes, lighthouse is just a command line program (it's making http requests under the hood) so it won't return the http response code from the underlying url it's checking, it just generates a JSON that either contains "no error" or details about the error in plain text. There's a variety of different possible errors but only one "success" message, which is "no error"